Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -498,6 +498,16 @@ static std::variant<UniqueRef<LocalFrameLoaderClient>, UniqueRef<RemoteFrameClie
     RELEASE_ASSERT_NOT_REACHED();
 }
 
+static RefPtr<Frame> frameOpener(std::optional<FrameIdentifier> identifier)
```

Comment:
Calling this `frameOpener` feels wrong. It looks up an identifier and returns the Frame for it, if it exists. Calling it `frameFromIdentifier` or something might be better.

Reply:
frameFromIdentifier is a much better name.  Changing WebProcess::webFrame should probably be done in a separate PR.  It looks like 20 of the 63 call sites would benefit.